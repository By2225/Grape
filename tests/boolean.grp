fun Int main() {
    Bool a;
    a = Yes;
    if(a) {
        print("Bool works");
    }
    return 0;
}
