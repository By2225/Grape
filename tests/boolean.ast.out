
fun Int main() {
Bool a;
a = true;
if (a)
{
print("Bool works");
}
return 0;
}
